src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -34,6 +34,7 @@
       <OutputItemType>Content</OutputItemType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
+      <OSGroup>$(OSGroup)</OSGroup>
```

Comment:
This is passing the $(OSGroup) variable that was passed to this project to the reference project; it appears it isn't passed automatically since I stopped getting an error after adding this